### DISCUSSION TITLE
a32nx/New Fly Pad2

### DISCUSSION THREAD

**Discussion by Lanka512** [flybywiresim/aircraft#5354] [Help] [answered]
Despite all the updates I still have the old flyPad in the development version .
How can I have the latest flyPadOS v2 ?
In my community folder A32NX: version 0.7.0-dev is what I have.
Cheers

**Comment by ExampleWasTaken** [comment, marked as answer]
Hi, use the [installer](https://api.flybywiresim.com/installer) to update your aircraft.

**Comment by Jonosay** [comment]
I had the same problem - using the updater did not update it - downloading from Github and dropping into community folder did not work, no matter what I tried, I still had the older version of the FlyPad.  

Then it occurred to me that I had originally "purchased" it from the Marketplace.  So I went to content manager in FS and deleted the plane from there.  Then I went to the community folder and deleted the folder that was still there to be sure of a completely fresh install and reinstalled it though the FBW installer.  Now I have the Flypad V2.  Hopefully this helps you as well!
